source/h2_connection.c

```diff
@@ -300,8 +300,11 @@ static struct aws_h2_connection *s_connection_new(
     connection->base.http_version = AWS_HTTP_VERSION_2;
     /* Init the next stream id (server must use even ids, client odd [RFC 7540 5.1.1])*/
     connection->base.next_stream_id = (server ? 2 : 1);
+    /* Stream window management */
     connection->base.manual_window_management = manual_window_management;
```

Comment:
trivial: it's a lot of changes, but worth renaming the internal variable ~base.manual_window_management~ -> `base.stream_manual_window_management`?